lib/common.css

```diff
@@ -197,3 +197,13 @@ button.goog-buttonset-default:hover {
   color: <<FORE_SECONDARY>>;
 }
 /** Update End **/
+
+/** Fixes Playlist button **/
+.material .sj-play-button-0 #buttonContent.sj-play-button {
+    background: <<FORE_SECONDARY>>;
+}
+
+.material .sj-play-button-0 #pulse.sj-play-button {
+    background: <<FORE_SECONDARY>>;
+    opacity: .3;
```

Comment:
This is simulating a different color right?  Instead of opacity we should probably use our `ALPHA` function so that the entire button doesn't become transparent, just the background color.

https://github.com/gmusic-utils/gmusic-theme.js/blob/master/src/gmusic-theme.js#L189